src/app/routes/utils/constructPageFetchUrl/index.test.ts

```diff
@@ -96,6 +96,9 @@ describe('constructPageFetchUrl', () => {
     ${LIVE_PAGE}    | ${null}         | ${null}    | ${'local'}  | ${'c0000000000t'}                                     | ${'http://localhost/api/local/ukrainian/live/c0000000000t'}
     ${LIVE_PAGE}    | ${null}         | ${null}    | ${'test'}   | ${'c0000000000t'}                                     | ${'https://mock-bff-path/?id=c0000000000t&service=ukrainian&pageType=live&serviceEnv=test'}
     ${LIVE_PAGE}    | ${null}         | ${null}    | ${'live'}   | ${'c0000000000t'}                                     | ${'https://mock-bff-path/?id=c0000000000t&service=ukrainian&pageType=live&serviceEnv=live'}
+    ${LIVE_PAGE}    | ${'arabic'}     | ${null}    | ${'local'}  | ${'/arabic/live/67574192'}                            | ${'https://mock-bff-path/?id=%2Farabic%2Flive%2F%2Farabic%2Flive%2F67574192&service=arabic&pageType=live&serviceEnv=local'}
```

Comment:
We also want integration tests for this.  Makes sure the local routing works.  Grab the data for the arabic live page, put into fixture data, assert that we can render the arabic live page. Then next pieces of work we can decide whether to add more cypress tests on the next js work.

Reply:
Integration tests added.